test/powershell/engine/Remoting/RemoteSession.Basic.Tests.ps1

```diff
@@ -10,47 +10,225 @@ Describe "New-PSSession basic test" -Tag @("CI") {
     }
 }
 
-Describe "JEA session Transcprit script test" -Tag @("Feature", 'RequireAdminOnWindows') {
-    It "Configuration name should be in the transcript header" -Pending {
+Describe "JEA session Transcript script test" -Tag @("Feature", 'RequireAdminOnWindows') {
+    BeforeAll {
+        Enable-PSRemoting -SkipNetworkProfileCheck
+    }
+
+    It "Configuration name should be in the transcript header" {
         [string] $RoleCapDirectory = (New-Item -Path "$TestDrive\RoleCapability" -ItemType Directory -Force).FullName
         [string] $PSSessionConfigFile = "$RoleCapDirectory\TestConfig.pssc"
         [string] $transScriptFile = "$RoleCapDirectory\*.txt"
         try
         {
             New-PSSessionConfigurationFile -Path $PSSessionConfigFile -TranscriptDirectory $RoleCapDirectory -SessionType RestrictedRemoteServer
-            Register-PSSessionConfiguration -Name JEA -Path $PSSessionConfigFile -Force -ErrorAction SilentlyContinue 
+            Register-PSSessionConfiguration -Name JEA -Path $PSSessionConfigFile -Force -ErrorAction SilentlyContinue
             $scriptBlock = {Enter-PSSession -ComputerName Localhost -ConfigurationName JEA; Exit-PSSession}
-            & $scriptBlock
+            # Invoke the script block in a different PowerShell instance to that when TestDrive tries to delete $RoleCapDirectory,
+            # the transcription has finished and the files are not locked.
+            [powershell]::Create().AddScript($scriptBlock).Invoke()
             $headerFile = Get-ChildItem $transScriptFile | Sort-Object LastWriteTime | Select-Object -Last 1
             $header = Get-Content $headerFile | Out-String
-            $header | Should BeLike "Configuration Name: JEA"
+            $header | Should Match "Configuration Name: JEA"
         }
         finally
         {
             Unregister-PSSessionConfiguration -Name JEA -Force -ErrorAction SilentlyContinue
         }
     }
-    
-}
 
+}
 
 Describe "JEA session Get-Help test" -Tag @("CI", 'RequireAdminOnWindows') {
```

Comment:
It takes about 10 seconds. I believe it is fast enough to be in CI. Thoughts?

Reply:
I am not sure but the testing guidelines say most CI tests take less than one second and the "Slow" tag should be considered if a test takes longer.  Since these are critical remoting functions it seems like we should leave them as CI tests, but probably should add the "Slow" tag.